namespaces/systemd.yml

```diff
@@ -0,0 +1,200 @@
+
+namespace:
+  name: systemd
+  type: group
+  description: >
+    Contains common fields specific to systemd journal
+  fields:
+  - name: CEE
```

Comment:
It seems this is just a general sub-namespace of sorts, and we should probably just let fields we don't know about land elsewhere.